models/init.go

```diff
@@ -35,7 +35,14 @@ func Init() {
 		case "UNSET", "sqlite", "sqlite3":
 			// 未指定数据库或者明确指定为 sqlite 时，使用 SQLite3 数据库
 			db, err = gorm.Open("sqlite3", util.RelativePath(conf.DatabaseConfig.DBFile))
-		case "mysql", "postgres", "mssql":
+		case "postgres":
+			db, err = gorm.Open(conf.DatabaseConfig.Type, fmt.Sprintf("host=%s user=%s password=%s dbname=%s port=%d sslmode=disable TimeZone=Asia/Shanghai",
```

Comment:
Is `TimeZone` a required parameter? if not, we'd better leave it to the default value.

Reply:
i will delete that